src/sync_client.rs

```diff
@@ -65,7 +66,8 @@ async fn process_block(
 	let commitment = block_body.header.extrinsics_root.commitment;
 	let block_num = block_body.header.number;
 	// now this is in `u64`
-	let positions = rpc::generate_random_cells(max_rows, max_cols);
+	let cell_count = -((1f64 - confidence / 100f64).log2()).ceil() as u32;
```

Comment:
The formula appears in two places. For better maintainability, can we put it into a separate function and call from two places?